VAMobile/src/components/FormWrapper/FormFields/VASelector.test.tsx

```diff
@@ -110,8 +109,8 @@ context('VASelector', () => {
       const checkBoxError = findByTestID(testInstance, 'CheckBoxError')
       expect(checkBoxError).toBeTruthy()
 
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews[textViews.length - 2].props.children).toEqual('ERROR MESSAGE')
+      expect(testInstance.findByProps({ children: 'ERROR MESSAGE' })).toBeTruthy()
```

Comment:
remove the test instance test and initialize correctly tests

Reply:
component and test instance should be removed from the file

Reply:
other unit tests should be updated to the screen.getByTestId or equivalent as test instance should no longer be in the file

Reply:
Updated.